app/ts/EffectsTable/EffectsTableCriteriaRows/EffectsTableDataSourceRow/ValueCell/EffectValueCell/EffectValueCellService.ts

```diff
@@ -37,44 +42,65 @@ function renderEnteredValues(effect: Effect, usePercentage: boolean): string {
   }
 }
 
-function getStringForValue(value: number, usePercentage: boolean): string {
```

Comment:
kind of two-minded file with half the functions being 'get' without retrieving and the other half 'render' without creating DOM elements directly :D